Index: frontend/src/app/router/router.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import { createBrowserRouter } from \"react-router\";\nimport { AuthPage } from \"@/pages/auth\";\nimport { HomePage } from \"@/pages/home\";\nimport { NotFoundPage } from \"@/pages/404\";\nimport { AppShell } from \"@/app/AppShell\";\nimport { AuthProvider } from \"@/features/auth\";\nimport { AppRoute } from \"@/shared/config\";\nimport { PrivateRoute } from \"./PrivateRoute\";\n\nexport const router = createBrowserRouter([\n  {\n    path: AppRoute.AUTH,\n    element: (\n      <AuthProvider>\n        <AuthPage />\n      </AuthProvider>\n    ),\n  },\n  {\n    path: AppRoute.ROOT,\n    element: (\n      <AuthProvider>\n        <PrivateRoute />\n      </AuthProvider>\n    ),\n    children: [\n      {\n        element: <AppShell />,\n        children: [\n          {\n            index: true,\n            element: <HomePage />,\n          },\n        ],\n      },\n    ],\n  },\n  {\n    path: AppRoute.NOT_FOUND,\n    element: <NotFoundPage />,\n  },\n]);\n
===================================================================
diff --git a/frontend/src/app/router/router.tsx b/frontend/src/app/router/router.tsx
--- a/frontend/src/app/router/router.tsx	(revision e0797dbbfee7203e1c450648a1ce55bdd60f9198)
+++ b/frontend/src/app/router/router.tsx	(date 1752055040110)
@@ -17,12 +17,12 @@
     ),
   },
   {
-    path: AppRoute.ROOT,
-    element: (
-      <AuthProvider>
-        <PrivateRoute />
-      </AuthProvider>
-    ),
+    // path: AppRoute.ROOT,
+    // element: (
+    //   <AuthProvider>
+    //     <PrivateRoute />
+    //   </AuthProvider>
+    // ),
     children: [
       {
         element: <AppShell />,
